octobot_trading/exchanges/implementations/margin_ccxt_exchange.py

```diff
@@ -13,165 +13,11 @@
-        return self.connector.get_trade_fee(symbol, order_type, quantity, price, taker_or_maker)
-
-    def get_fees(self, symbol):
-        return self.connector.get_fees(symbol)
-
-    def get_pair_from_exchange(self, pair) -> str:
-        return self.connector.get_pair_from_exchange(pair)
-
-    def get_split_pair_from_exchange(self, pair) -> (str, str):
-        return self.connector.get_split_pair_from_exchange(pair)
-
-    def get_exchange_pair(self, pair) -> str:
-        return self.connector.get_exchange_pair(pair)
-
-    def get_pair_cryptocurrency(self, pair) -> str:
-        return self.connector.get_pair_cryptocurrency(pair)
-
-    def get_default_balance(self):
-        return self.connector.get_default_balance()
-
-    def get_rate_limit(self):
-        return self.connector.get_rate_limit()
-
-    async def switch_to_account(self, account_type: enums.AccountTypes):
-        return await self.connector.switch_to_account(account_type=account_type)
-
-    def parse_balance(self, balance):
-        return personal_data.parse_decimal_portfolio(self.connector.parse_balance(balance))
-
-    def parse_trade(self, trade):
-        return self.connector.parse_trade(trade)
-
-    def parse_order(self, order):
-        return self.connector.parse_order(order)
-
-    def parse_ticker(self, ticker):
-        return self.connector.parse_ticker(ticker)
-
-    def parse_ohlcv(self, ohlcv):
-        return self.connector.parse_ohlcv(ohlcv)
-
-    def parse_order_book(self, order_book):
-        return self.connector.parse_order_book(order_book)
-
-    def parse_order_book_ticker(self, order_book_ticker):
-        return self.connector.parse_order_book_ticker(order_book_ticker)
-
-    def parse_timestamp(self, data_dict, timestamp_key, default_value=None, ms=False):
-        return self.connector.parse_timestamp(data_dict, timestamp_key, default_value=default_value, ms=ms)
-
-    def parse_currency(self, currency):
-        return self.connector.parse_currency(currency)
-
-    def parse_order_id(self, order):
-        return self.connector.parse_order_id(order)
-
-    def parse_order_symbol(self, order):
-        return self.connector.parse_order_symbol(order)
-
-    def parse_status(self, status):
-        return self.connector.parse_status(status)
-
-    def parse_side(self, side):
-        return self.connector.parse_side(side)
-
-    def parse_account(self, account):
-        return self.connector.parse_account(account)
-
-    def clean_recent_trade(self, recent_trade):
-        return self.connector.clean_recent_trade(recent_trade)
-
-    def clean_trade(self, trade):
-        return self.connector.clean_trade(trade)
 
-    def clean_order(self, order):
-        return self.connector.clean_order(order)
+class MarginCCXTExchange(ccxt_exchange_commons.CCXTExchangeCommons,
+                         exchanges_types.MarginExchange):
+    def get_default_type(self):
+        return 'margin'
```

Comment:
you have to put a new line here (end of file)

Reply:
done